compass/ocean/tests/sphere_transport/rotation_2d/init.py

```diff
@@ -0,0 +1,46 @@
+from compass.model import run_model
+from compass.step import Step
+
+
+class Init(Step):
+    """
+    A step for an initial condition for for the cosine bell test case
+    """
+
+    def __init__(self, test_case, resolution):
+        """
+        Create the step
+
+        Parameters
+        ----------
+        test_case :
+            compass.ocean.tests.global_convergence.rotation_2d.Rotation2D
```

Comment:
Same comment about line length -- this needs to go back to being one line.
And again, the description got chopped in the last commit